sources/packages/backend/apps/queue-consumers/src/queue-consumers.module.ts

```diff
@@ -50,13 +100,55 @@ import { ATBCIntegrationModule } from "@sims/integrations/atbc-integration";
     SFASIntegrationScheduler,
     ATBCResponseIntegrationScheduler,
     StudentAssessmentService,
+    StudentsAssessmentService,
     SshService,
     QueueService,
-    DisbursementScheduleService,
     CRAResponseIntegrationScheduler,
     CRAProcessIntegrationScheduler,
     SequenceControlService,
     WorkflowClientService,
+    SINValidationProcessIntegrationScheduler,
```

Comment:
It seems that there are way too many providers/classes imported here.
It is not the case when the `queue-consumers` should be importing more modules and fewer providers?